lib/datasource/index.spec.ts

```diff
@@ -44,9 +44,20 @@ describe(getName(__filename), () => {
       }
       return true;
     }
+    function filterClassBasedDatasources(name: string): boolean {
+      return !['cdnjs', 'clojure'].includes(name);
+    }
```

Comment:
better use `getDatasources()` and check `obj instanceOf Datasource`, so we do not need to change the test on every datasource migration.

Reply:
Will come back to this later.

I can't directly modify `datasource.getDatasources()` and `clone` is returning `{}` for some reason 🤔

Reply:
clone with `new Map(datasource.getDatasources())` ? 😏